cs104/server.go

```diff
@@ -144,3 +144,8 @@ func (sf *Server) Params() *asdu.Params { return &sf.params }
 
 // UnderlyingConn imp interface Connect
 func (sf *Server) UnderlyingConn() net.Conn { return nil }
+
+// LoadCurrentTimeZone current time zone to server
+func (sf *Server) LoadCurrentTimeZone() {
```

Comment:
Is this api really needed ??

if need write like follow is best:

```
// SetInfoObjTimeZone set info object time zone  
func (sf *Server) SetInfoObjTimeZone(zone *time.Location) {
	sf.params.InfoObjTimeZone = zone
}
```

Reply:
No its not necessary. I though it was just a decent way to set the time object. Your version is good too.